src/test/groovy/com/netflix/nebula/lint/rule/dependency/UnusedDependencyRuleSpec.groovy

```diff
@@ -468,31 +468,6 @@ class UnusedDependencyRuleSpec extends TestKitSpecification {
         !results.output.contains('unused-dependency')
     }
 
-    @Issue('46')
-    def 'remove unused compileOnly dependencies'() {
```

Comment:
This will be broken again, but because we are not using this rule I think benefits overweight it.